File: Practicas/Yahir-Hermosillo/Parcial3/Practica6/SortingHat.java
import java.util.Random;

public class SortingHat {
    private static final String[] HOUSES = {
        "Gryffindor", "Slytherin", "Ravenclaw", "Hufflepuff"
    };

    public void assignHouse(MagicPerson person) {
        Random random = new Random();
        String house = HOUSES[random.nextInt(HOUSES.length)];
        person.setHouse(house);
        System.out.println("¡" + person.getName() + ", serás de... " + house.toUpperCase() + "!");
    }
}

File: Practicas/Marco-Oregel/Practica6/SortingHat.py
import random
from Student import Student
from MagicPerson import MagicPerson

class SortingHat:
    def __init__(self):
        pass

    def assignHouse(self, student):
        houses = ["Gryffindor","Slytherin","Ravenclaw", "Hufflepuff"]
        house = random.choice(houses)
        student.house = house
        print(f"{student.name}, seras de... {house}!")


    
    